### ISSUE TITLE
Download Folder produces empty files on remote !!

### ISSUE THREAD

**Issue by kgansberger** [liximomo/vscode-sftp#390]
**Do you read the FAQ?**
- [x] Yes.
- [ ] [I am going to read now.](https://github.com/liximomo/vscode-sftp/blob/master/FAQ.md)

**Describe the bug**
when i download folder with context menu or sftp:Download Active Folder or sftp:Download Project a few files on remote are 0kb. 

**To Reproduce**
as above

**Expected behavior**

same folder and file structur remote -> local

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Mac]
 - VSCode Version [e.g.  1.28.2]
 - Extension Version [e.g. 1.7.3]

**Extension Logs** - *required*

http://cl.ly/af4955c48e3e

when i look in ftp i see that files are written new ( some are for short time 0kb.). error not every time ....

**Comment by kgansberger** [comment]
PS: perhaps its 
```
 "watcher": {
            "files": "**",
            "autoUpload": true, //here is probably the problem
            "autoDelete": true
        }```

**Comment by liximomo** [comment]
Yes. You can't watcher anything. This will cause an upload while a download begins.

**Comment by liximomo** [comment]
This may be fixed in v1.7.4.
